### PR TITLE
As a part of 1.18 docs release work updating the config.toml

### PR DESCRIPTION
**Pull request by VineethReddy02**
This PR updates config.toml for the release-1.17 branch.

### CODE REVIEW

**Comment by sftim** [comment]
Helps with issue #19841

File: config.toml [hunk 1]
```diff
@@ -66,12 +66,12 @@ time_format_blog = "Monday, January 02, 2006"
...
+latest = "v1.18"
 
-fullversion = "v1.17.0"
+fullversion = "v1.17.4"
 version = "v1.17"
-githubbranch = "master"
-docsbranch = "master"
+githubbranch = "v1.17.4"
+docsbranch = "release-1.17"
 deprecated = false
```
Comment (kbhawkey) [inline config.toml]: Change `deprecated` to `true`?

Here is a snippet from 1.16 config.toml:
```
fullversion = "v1.16.8"
version = "v1.16"
githubbranch = "v1.16.8"
docsbranch = "release-1.16"
deprecated = true
currentUrl = "https://kubernetes.io/docs/home/"
nextUrl = "https://kubernetes-io-vnext-staging.netlify.com/"
githubWebsiteRepo = "github.com/kubernetes/website"
githubWebsiteRaw = "raw.githubusercontent.com/kubernetes/website"
```

**Comment by VineethReddy02** [comment]
@kbhawkey Made requested changes :)

**Comment by kbhawkey** [comment]
Preview:
https://deploy-preview-19838--k8s-v1-17.netlify.com/

/lgtm

File: config.toml [hunk 2]
```diff
@@ -84,10 +84,10 @@ announcement = false
...
 
 [[params.versions]]
-fullversion = "v1.17.0"
-version = "v1.17"
-githubbranch = "v1.17.0"
-docsbranch = "release-1.17"
+fullversion = "v1.18.0"
+version = "v1.18"
+githubbranch = "v1.18.0"
+docsbranch = "release-1.18"
```
Comment (kbhawkey) [inline config.toml]: @VineethReddy02 , What happened to the `params.pushAssets`?
Does this section belong after line 85?

**Comment by VineethReddy02** [comment]
@kbhawkey I have placed/indented push.Assets in 85 line as it was before.
Can we have ```lgtm``` and ```approve``` for this?

File: config.toml (hunk 2 above)
Comment (VineethReddy02) [reply config.toml]: I just changed the placing of push. Assets. I just put it back to the same line now.

**Pull request merged**